udata_front/theme/gouvfr/templates/participez/participez.html

```diff
@@ -1,4 +1,4 @@
-<div class="bg-blue-350">
+<div class="bg-blue-350 text-white">
```

Comment:
I don't think it's needed, there are already `text-white` and `btn-secondary-white` classes used.

Reply:
I don't know why I added it